### PR TITLE
Pin versions pulled in from socrata-http

### PR DESCRIPTION
**Pull request by kaidamasaki**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -10,16 +10,29 @@ resolvers ++= Seq(
...
 libraryDependencies ++= Seq(
   "ch.qos.logback"           % "logback-classic"          % "1.1.2",
+  "com.rojoma"              %% "rojoma-json-v3"           % "3.3.0",
+  "com.rojoma"              %% "rojoma-json-v3-jackson"   % "1.0.0"
+    exclude("com.rojoma", "rojoma-json-v3_2.10"),
   "com.rojoma"              %% "simple-arm-v2"            % "2.1.0",
-  "com.socrata"             %% "socrata-http-client"      % "3.3.0",
-  "com.socrata"             %% "socrata-http-jetty"       % "3.3.0",
+  "com.socrata"             %% "socrata-http-client"      % "3.3.0"
+    exclude("com.rojoma", "rojoma-json_2.10")
```
Comment (urmilan) [inline build.sbt]: How come you had to specify 2.10?

Comment (kaidamasaki) [inline build.sbt]: The excludes are handled by ivy, and it works based on the Java name of the package.

Comment (urmilan) [inline build.sbt]: Ugh. Apart from that LGTM :+1:

File: build.sbt [hunk 2]
```diff
@@ -10,16 +10,31 @@ resolvers ++= Seq(
...
+    exclude("com.rojoma", "rojoma-json-v3_2.10")
+    exclude("com.rojoma", "rojoma-json-v3-jackson_2.10")
+    exclude("com.rojoma", "simple-arm_2.10")
+    exclude("com.rojoma", "simple-arm-v2_2.10")
+    exclude("com.socrata", "socrata-thirdparty-utils_2.10"),
+  "com.socrata"             %% "socrata-http-jetty"       % "3.3.0"
+    exclude("com.rojoma", "rojoma-json_2.10")
```
Comment (velvia) [inline build.sbt]: Think there is some way to exclude everything from com.rojoma, if that's simpler.

**Comment by kaidamasaki** [comment]
Bump.

File: build.sbt [hunk 3]
```diff
@@ -52,13 +59,13 @@ Revolver.settings
...
 styleTask := { val _ = (scalastyle in Compile).toTask("").value }
-(Keys.`package` in Compile) <<= (Keys.`package` in Compile) dependsOn styleTask
+  (Keys.`package` in Compile) <<= (Keys.`package` in Compile) dependsOn styleTask
 AssemblyKeys.assembly <<= AssemblyKeys.assembly dependsOn styleTask
 
 // Make "test:test" depend on "test:scalastyle"
 lazy val testStyleTask = taskKey[Unit]("a task that wraps 'test:scalastyle' with no input parameters.")
 testStyleTask := { val _ = (scalastyle in Test).toTask("").value }
-(test in Test) <<= (test in Test) dependsOn (testStyleTask)
+  (test in Test) <<= (test in Test) dependsOn (testStyleTask)
```
Comment (sokrahta) [inline build.sbt]: whitespace nit: why did lines 62 and 68 add an indent?

File: build.sbt [hunk 4]
```diff
@@ -10,16 +10,23 @@ resolvers ++= Seq(
...
+    ExclusionRule(organization = "com.socrata", name = "socrata-thirdparty-utils_2.10")),
   "com.socrata"             %% "socrata-thirdparty-utils" % "3.0.0",
   "com.typesafe"             % "config"                   % "1.2.1",
   "commons-codec"            % "commons-codec"            % "1.10",
   "commons-io"               % "commons-io"               % "2.4",
   "no.ecc.vectortile"        % "java-vector-tile"         % "1.0.1",
-  "org.velvia"              %% "msgpack4s"                % "0.4.3",
-  "org.apache.curator"       % "curator-x-discovery"      % "2.7.0"
+  "org.apache.curator"       % "curator-x-discovery"      % "2.7.0",
+  "org.velvia"              %% "msgpack4s"                % "0.4.3"
```
Comment (sokrahta) [inline build.sbt]: does this ordering change anything?

File: build.sbt (hunk 3 above)
Comment (kaidamasaki) [inline build.sbt]: Not sure.  Emacs decided to do that when I re-indented the file.  I can un-indent them.

File: build.sbt (hunk 4 above)
Comment (kaidamasaki) [inline build.sbt]: Nope, they were just out of alphabetical order.

**Comment by sokrahta** [comment]
lgtm :shipit:

**Pull request merged**